README.md

```diff
@@ -346,6 +346,79 @@ allow = response {
 }
 ```
 
+## Example with JWT payload passed from Envoy
+
+Envoy can be configured to pass validated JWT payload data into the `ext_authz` filter with `metadata_context_namespaces` and `payload_in_metadata`.
+
+### Example Envoy Configuration
+```ruby
+http_filters:
+- name: envoy.filters.http.jwt_authn
+  typed_config:
+  "@type": type.googleapis.com/envoy.config.filter.http.jwt_authn.v2alpha.JwtAuthentication
+  providers:
+    example:
+      payload_in_metadata: verified_jwt
+      <...>
+- name: envoy.ext_authz
+  config:
+    metadata_context_namespaces:
+    - envoy.filters.http.jwt_authn
+    <...>
+```
```

Comment:
Is this configuration applicable to the `EnvoyFilter` kind  ? Or does it need to be updated ?